packages/flutter_tools/lib/src/run_cold.dart

```diff
@@ -168,6 +168,10 @@ class ColdRunner extends ResidentRunner {
 
   @override
   Future<void> cleanupAtFinish() async {
+    for (FlutterDevice flutterDevice in flutterDevices) {
```

Comment:
We should have a test here to ensure this is called

Reply:
sgtm

Reply:
done, PTAL @jonahwilliams